Sources/Extensions/UIKit/UIViewControllerExtensions.swift

```diff
@@ -77,7 +77,27 @@ public extension UIViewController {
         present(alertController, animated: true, completion: nil)
         return alertController
     }
-
+    
+    /// SwifterSwift: Helper method to add a UIViewController as a childViewController.
+    ///
+    /// - Parameters:
+    ///   - child: the view controller to add as a child
+    public func add(_ child: UIViewController) {
+        addChildViewController(child)
+        view.addSubview(child.view)
+        child.didMove(toParentViewController: self)
+    }
+    
+    /// SwifterSwift: Helper method to remove a UIViewController from its parent.
+    public func remove() {
+        guard parent != nil else {
+            return
```

Comment:
Can you move the `return` statement to the line above?

Reply:
sure!